CHANGELOG.md

```diff
@@ -4,10 +4,18 @@ All notable changes to this project are documented below.
 
 The format is based on [keep a changelog](http://keepachangelog.com/) and this project uses [semantic versioning](http://semver.org/).
 
-## [Unreleased]
+## [1.1.0] - 2023-11-29
 #### Changed
-- Complete client update to support Nakama 2.0 and Nakama 3.0
-- Update for Swift 5.
+- Expose expiredTimeSpan in client initializer.
+- Change Int32 and Int64 parameters to Int.
+- Add: listFriends, deleteFriends, updateAccount, listChannelMessages, and listMatches.
```

Comment:
Sorry minor nitpick, can you rearrange this to:

 ```
#### Added
- listFriends, deleteFriends, updateAccount, listChannelMessages, and listMatches.
```

Reply:
Done.